src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,117 @@
+
+We recommend updating your synthetic monitors that were created using the legacy runtime to take advantage of these capabilities. We plan to release this new runtime for private monitoring locations later this year.
+
+<Callout variant="important">
+Before switching, make sure that you've updated your allow lists to accept network requests from the new runtime’s [public IP ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
+</Callout>
+
+## Convert your monitors to the new runtime [#runtime-convert]
+
+Existing monitors default to the runtime used at their time of creation. To prevent your critical monitors from breaking during future end of life processes, we recommend converting your public monitors sooner rather than later. To convert:
+
+1. Go to [one.newrelic.com](https://one.newrelic.com/nr1-core), click **Synthetics**, then select the monitor you want to edit.
+2. From the **Settings** tab, click **General**. 
+3. Use the dropdown menu to switch the current runtime view (use Node 16 for scripted API monitors and Chrome 100 for scripted browser monitors). 
+4. Click **Validate** to check that your monitors function in the new runtime. Make any script script modifications if needed. 
+5. Save.
+
+## Switch to the new runtime from the UI [#runtime-UI]
+
+From the UI, you have the option to toggle between the legacy runtime and the new runtime. While the new runtime is backwards compatible with the legacy runtime, we recommend that you convert to the new environment as soon as possible. 
+
+1. Go to [one.newrelic.com](https://one.newrelic.com/nr1-core), then click **Synthetics**.
+2. Click **Create monitor**, then select the public monitor type you want to create.
+3. Use the **Runtime** dropdown menu to switch between the legacy and new runtime environment. 
+4. Create your monitor. 
+
+## Use NerdGraph to manage your monitors
+
+We have expanded support for synthetic monitors to NerdGraph. By using NerdGraph to manage your monitors, you get: 
+
+* Management of your certificate check monitors, broken links monitors, and step monitors, not previously supported through REST APIs.
+* Simplified processes for creating scripted monitors.
+* The ability to attach scripts to scripted monitors with only one call, rather than the four calls needed with REST APIs.
+* The ability to add tags to your monitors. 
+
+Unlike with REST APIs, NerdGraph lets you programatically create [broken links](#create-broken-links), [step](#create-step), and [certificate check](create-certificate-check) monitors.
+
+## Get custom timing details with your scripted API monitors
+
+Previously, you needed to use `$util.insights.set` to time and trend actions across your monitor executions, but the requests would not generate any data in the New Relic UI. This prevented you from using queries and dashboards to extract values from these requests. 
```

Comment:
Previously, use cases for the scripted API monitor outside of using the `$http` object would result in only the pass / fail status and script log being available in the New Relic UI. No timing details were available. Logic could be added to time requests and those timers could be reported to the SyntheticCheck event using `$util.insights.set`, but visibility was still limited in the check result.

The new synthetic monitoring runtime still includes out-of-the-box timing details when using the default `$http` object. Additionally, custom timing details can be added by [using $har](/docs/synthetics/synthetic-monitoring/scripting-monitors/custom-timing-details).